### PR TITLE
Branch 1

### PR DESCRIPTION
**Pull request by amklass11**
# ACHIEVEMENTS
All requirments from the exercise have been done
1. files added 
2. linter added 
3. neccessary files were added to gitignore
4. hello microverse text added to index.html

PULL REQUEST OPEN

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+# Show your support
+Give a ⭐️ if you like this project! and cheers✨
+
+ # Acknowledgments
+Microverse offered guidance on how to make the project more professional
+
+
+# 📝 License
+This Project is MIT licensed.
```
Comment (Whoistolu) [inline README.md]: - [ ] [REQUIRED] Kindly check the link as it is currently broken. We should have a link to your GitHub, Twitter, and Linkedin.
Please add a description to this project.
Also, ensure that the "Getting started" section clearly states sets of instructions to follow in order to get a copy of this project to another local machine.
Please use [this template](https://github.com/microverseinc/readme-template) for proper guidance.
The README file is the page that gives a new user an idea of what your project entails, so, let's make it more detailed and professional. 👍🏽

File: INDEX.HTML [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<html>
+
+
+<body>Hello Microverse!</body>
+
+</html>
```
Comment (Whoistolu) [inline INDEX.HTML]: - [x] [REQUIRED] It looks like the `head` tag and its components are missing here. The Meta charset attribute specifies the character encoding for the HTML document. Kindly add in order to fix the `Lighthouse` errors currently on the linters.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @amklass11,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

To Highlight:
- [ ] [REQUIRED] Please let's et up linters correctly as it currently throws errors. Linter result should be green. Linter should report NO errors before asking for a review.
- [ ] The `style.css` file is empty so it obviously throws a `stylelint` error. Kindly add some styles to avoid errors.
 Linters provide an objective and measurable assessment of code quality.


_Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @Whoistolu if you have any comments.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: INDEX.HTML (hunk 2 above)
Comment (amklass11) [reply INDEX.HTML]: I did that, however the result is still an error

Comment (Whoistolu) [reply INDEX.HTML]: Hi @amklass11, below is the report of the stylelint error. Try to fix them all and you should be fine.
Screenshot from 2022-02-16 08-50-36
If you also want to see the errors, please click on details right in front of the linters on this PR.

Comment (amklass11) [reply INDEX.HTML]: okay thank you what about the light house error

Comment (Whoistolu) [reply INDEX.HTML]: Kindly implement the meta charset attribute. It should fix it.

Comment (amklass11) [reply INDEX.HTML]: Hello, My apologies meta charset was included however error still persist

File: INDEX.HTML [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Hello Microverse</title>
+        <link rel="stylesheet" href="style.css">
+        <title>hello hello</title>
```
Comment (Wes-Isaac) [inline INDEX.HTML]: - [x] You have the title tag twice which might be causing the lighthouse error please remove one of them.
- [x] Please fix the lighthouse linter error.

image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @amklass11, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: INDEX.HTML (hunk 3 above)
Comment (amklass11) [reply INDEX.HTML]: Hello I'm aware of the linter error,
i'm asking how to fix it

Comment (amklass11) [reply INDEX.HTML]: this has been a problem I've been  dealing with for sometime

Comment (amklass11) [reply INDEX.HTML]: Its done but the error persist

**Review by Deepakdanger — APPROVED** [review]
Hi @amklass11, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**